$NetBSD: patch-unix_Makefile,v 1.3 2017/11/29 16:34:40 hauke Exp $

cpp(1) only accepts one file for input, we want cc -E here.

--- unix/Makefile.orig	2017-08-23 01:50:11.000000000 +0000
+++ unix/Makefile
@@ -73,6 +73,7 @@ LDFLAGS_ARCH = -Wl,-map,$@.map -Wl,-dead
 else
 # Use gcc syntax for map file
 LDFLAGS_ARCH = -Wl,-Map=$@.map,--cref -Wl,--gc-sections
+CPP = $(CC) -E
 endif
 LDFLAGS = $(LDFLAGS_MOD) $(LDFLAGS_ARCH) -lm $(LDFLAGS_EXTRA)
 
